packages/rpc-parsed-types/src/bpf-upgradeable-loader-accounts.ts

```diff
@@ -3,16 +3,16 @@ import type { Base64EncodedDataResponse, Slot } from '@solana/rpc-types';
 
 import type { RpcParsedType } from './rpc-parsed-type';
 
-type ProgramAccount = Readonly<{
+type JsonParsedProgramAccount = Readonly<{
     programData: Address;
 }>;
 
-type ProgramDataAccount = Readonly<{
+type JsonParsedProgramDataAccount = Readonly<{
```

Comment:
`JsonParsedBpfUpgradeableLoaderProgramAccount` & `JsonParsedBpfUpgradeableLoaderProgramDataAccount`?

It's a mouthful, but these might confuse people into thinking they're the parent types of `JsonParsedXProgramAccount`, or for a generic `ProgramAccount` altogether.

Reply:
`JsonParsedBpfUpgradeableLoaderProgramAccount` clashes with the tagged union type

I'll just add `Bpf` to these to make sure it's clear what they come from. These nested types aren't exported currently, so I'd rather lose a bit of consistency there than on the exported tagged union type.